### ISSUE TITLE
NoAttributeError on gcloud sql command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#190]
```
What steps will reproduce the problem?


What is the expected output? What do you see instead?


Please provide any additional information below.


The output of 'gcloud info':

Google Cloud SDK [0.9.71]

Platform: [Linux, x86_64]
Python Version: [2.7.6 (default, Jun 22 2015, 17:58:13)  [GCC 4.8.2]]
Site Packages: [Disabled]

Installation Root: [/home/ubiquitoust/google-cloud-sdk]
Installed Components:
  core: [2015.07.24]
  core-nix: [2015.06.02]
  gcloud: [2015.07.24]
  gsutil-nix: [4.12]
  beta: [2015.07.16]
  gsutil: [4.13]
  bq: [2.0.18]
  bq-nix: [2.0.18]
System PATH: 
[/home/ubiquitoust/google-cloud-sdk/bin:/home/ubiquitoust/.rvm/gems/ruby-2.1.5/b
in:/home/ubiquitoust/.rvm/gems/ruby-2.1.5@global/bin:/home/ubiquitoust/.rvm/rubi
es/ruby-2.1.5/bin:$HOME/jdk1.7.0_76/bin:$HOME/android-studio/bin:$HOME/Android/S
dk/build-tools:$HOME/Android/Sdk/platform-tools:$HOME/Android/Sdk/tools:/usr/loc
al/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
:/home/ubiquitoust/.rvm/bin:/home/ubiquitoust/.rvm/bin:/usr/lib/postgresql/9.3/b
in:/usr/local/ec2/ec2-api-tools-1.7.3.0/bin:/usr/local/apache-maven/apache-maven
-3.1.1/bin]
Cloud SDK on PATH: [True]

Installation Properties: [/home/ubiquitoust/google-cloud-sdk/properties]
User Config Directory: [/home/ubiquitoust/.config/gcloud]
User Properties: [/home/ubiquitoust/.config/gcloud/properties]
Current Workspace: [None]
Workspace Config Directory: [None]
Workspace Properties: [None]

Account: [scotthansontodd@gmail.com]
[output truncated]

Project: [advance-copilot-93309]

Current Properties:
  [core]
    project: [advance-copilot-93309]
    account: [scotthansontodd@gmail.com]
    disable_usage_reporting: [False]

Logs Directory: [/home/ubiquitoust/.config/gcloud/logs]
Last Log File: 
[/home/ubiquitoust/.config/gcloud/logs/2015.08.04/22.40.34.824727.log]


```

Original issue reported on code.google.com by `scotthan...@gmail.com` on 5 Aug 2015 at 5:47

Attachments:
- [22.40.34.824727.log](https://storage.googleapis.com/google-code-attachments/google-cloud-sdk/issue-190/comment-0/22.40.34.824727.log)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. It appears to be a bug. Created internal issue 
(22964797).

What command were you trying to run when you got this error?


```

Original comment by `che...@google.com` on 5 Aug 2015 at 12:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report!
We fixed api client override for sql in version 0.9.72. Please update to this 
version and it should work ok. If it still doesn't work ok after updating, 
please reinstall Cloud SDK.
```

Original comment by `svalen...@google.com` on 5 Aug 2015 at 5:49
- Changed state: **Fixed**
